### PR TITLE
Fix Doc Build

### PR DESCRIPTION
**Pull request by awholegunch**
Fixes a broken Doc Build.
Removed Javadoc links, as we don't run Javadocs on these files.
Updated copyrights dates. Sync'd the example and the archetype.
Updated the MD5 hash to fix the build.

Running as a Quick Build: http://builds.cask.co/browse/CDAP-DQB247-2 (passes!)
Merged develop back in and started a new build: http://builds.cask.co/browse/CDAP-DQB247-3 (passes)

### CODE REVIEW

File: cdap-archetypes/cdap-app-archetype/src/main/resources/archetype-resources/src/main/java/HelloWorld.java [hunk 1]
```diff
@@ -98,7 +98,7 @@ public void process(StreamEvent event) {
   }
 
   /**
-   * A {@link Service} that creates a greeting using a user's name.
+   * A Service that creates a greeting using a user's name.
```
Comment (Denton-L) [inline cdap-archetypes/cdap-app-archetype/src/main/resources/archetype-resources/src/main/java/HelloWorld.java]: Maybe change `@link` to `@code`? Same with below.

Comment (bdmogal) [inline cdap-archetypes/cdap-app-archetype/src/main/resources/archetype-resources/src/main/java/HelloWorld.java:101]: ``Service`` is a class in ``cdap-api``. We should have a javadoc for it, no?

File: cdap-examples/HelloWorld/src/main/java/co/cask/cdap/examples/helloworld/HelloWorld.java [hunk 2]
```diff
@@ -98,7 +98,7 @@ public void process(StreamEvent event) {
   }
 
   /**
-   * A {@link Service} that creates a greeting using a user's name.
+   * A Service that creates a greeting using a user's name.
```
Comment (bdmogal) [inline cdap-examples/HelloWorld/src/main/java/co/cask/cdap/examples/helloworld/HelloWorld.java:101]: ditto

File: cdap-examples/HelloWorld/src/main/java/co/cask/cdap/examples/helloworld/HelloWorld.java [hunk 3]
```diff
@@ -113,7 +113,7 @@ protected void configure() {
   }
 
   /**
-   * Greeting {@link Service} handler.
+   * Greeting Service handler.
```
Comment (bdmogal) [inline cdap-examples/HelloWorld/src/main/java/co/cask/cdap/examples/helloworld/HelloWorld.java:116]: ditto

File: cdap-archetypes/cdap-app-archetype/src/main/resources/archetype-resources/src/main/java/HelloWorld.java (hunk 1 above)
Comment (awholegunch) [reply cdap-archetypes/cdap-app-archetype/src/main/resources/archetype-resources/src/main/java/HelloWorld.java:101]: We do...but as we don't run Javadocs on the `examples`, having this in the examples just clutters up the comments. We haven't added links elsewhere in these examples to our Javadocs.

File: cdap-examples/HelloWorld/src/main/java/co/cask/cdap/examples/helloworld/HelloWorld.java (hunk 2 above)
Comment (awholegunch) [reply cdap-examples/HelloWorld/src/main/java/co/cask/cdap/examples/helloworld/HelloWorld.java:101]: Ibid.

File: cdap-examples/HelloWorld/src/main/java/co/cask/cdap/examples/helloworld/HelloWorld.java (hunk 3 above)
Comment (awholegunch) [reply cdap-examples/HelloWorld/src/main/java/co/cask/cdap/examples/helloworld/HelloWorld.java:116]: Ibid.

**Review by bdmogal — APPROVED** [review]

**Pull request merged**